test/unit/request.test.js

```diff
@@ -282,6 +282,46 @@ describe('Request', function () {
             var request = new Request();
             expect(request.getHeaders()).to.eql({});
         });
+
+        it('should handle duplicate headers using `multiValue` option', function () {
+            var rawRequest = {
+                    url: 'postman-echo.com',
+                    header: [{
+                        key: 'name',
+                        value: 'alpha'
+                    }, {
+                        key: 'name',
+                        value: 'beta'
+                    }, {
+                        key: 'name',
+                        value: 'gamma',
+                        disabled: true
+                    }]
+                },
+                request = new Request(rawRequest);
+            expect(request.getHeaders({ enabled: true, multiValue: true })).to.eql({
```

Comment:
Also add a test that checks for this being stringified correctly.

Reply:
This won't affect the `toString` flow, multiple `key-value` header pairs will be returned irrespective of the duplicates.